static/docs/user-guide/experiments/dirs.md

```diff
@@ -0,0 +1,97 @@
+# How to Manage Experiments by Directories
+
+Using a separate directory for each experiment is the most intuitive solution
+for managing experiments and is the first thing that comes to mind. Most of
+DS/ML practitioners are already familiar with this approach.
+
+<p align="center">
+<img src="/static/img/user-guide/experiments/dirs.png" />
+</p>
+
+This approach is most suitable when the different experiments that are being
+managed do not have significant differences in their code or the pipeline, but
+maybe change on the input datasets, processing parameters, configuration
+settings, etc.
+
+Often it is possible to generate these experiment directories automatically (or
+almost automatically) from the code of the main project (using the parameters or
+configuration settings), so keeping them in Git is not interesting or useful.
+What we would like to track instead are just the parameters that were used to
+generate the experiment directory and the results of the evaluation (metrics),
+so that we can figure out which parameters give the best results.
+
+## Examples
+
+There is a very basic example of using directories for each experiment at the
+end of
+[this interactive tutorial](https://katacoda.com/dvc/courses/basics/pipelines).
+
+## How it works
+
+If we have a directory named `experiment1/` which contains the pipeline of the
+first experiment, and we want to create another experiment on `experiment2/`,
+which is based on the first one, often it is as easy as:
+
+```dvc
+$ cp --reflink -R experiment1/ experiment2/
```

Comment:
reflinks are not supported on majority of systems. So, it's not that easy to copy it w/o hitting some performance problems by copying data. We should think a bit more on what should we do here.

Reply:
> reflinks are not supported on majority of systems

"reflink" is a filesystem feature, and all the systems support filesystems with reflink. For example in Linux there are XFS, Btrfs, ZFS etc. The option "--reflink" here is a hint or a reminder that they need to use a filesystem that supports reflinks (like XFS), which they should be doing anyway since the start of the project.

Reply:
see my bigger comment in the review. We can't rely on XFS, etc - for now 80-90% of systems are not configured to use them by default. And it won't change anytime soon. So, if we want to provide a meaningful solution to experiments management we should at least mention something about data, code and other things.

Reply:
> 80-90% of systems are not configured to use them by default

That's right, but they are not configured to use DVC by default either. Installing and using XFS is actually much easier than installing and using DVC.
What I am trying to say is that if someone is working on a data project, he can and should install and use XFS (or some other deduplicating filesystem). This is a basic requirement.

Reply:
It's very very far from the reality. There are a lot (majority?) of cases when you don't have a choice. Your argument applies only to freelancers and students.